week07/day04/FoxClub/Views/home/index.cshtml

```diff
@@ -0,0 +1,21 @@
+﻿@using FoxClub.Models;
+@model Fox;
+
+<html>
+<head>
+    <meta name="viewport" content="width=device-width" />
+    <title>@ViewBag.Title</title>
+</head>
+<body>
+    <ul>
+        <li><a href="~">Information</a></li>
+    </ul>
+    <img src="~/greenfox.png" />
+    <p>This is @Model.Name. Currently living on pizza and lemonade. He knows 2 tricks.</p>
```

Comment:
I'd start with `X` as a placeholder if the functionality does not exist yet. `He knows X tricks`